spec/requests/api/v4/questionnaire_sync_spec.rb

```diff
@@ -0,0 +1,106 @@
+require "rails_helper"
+
+RSpec.describe "Questionnaires sync", type: :request do
```

Comment:
Let's add a test the check only one version of the questionnaire is synced?

Reply:
Questionnaire controller queries questionnaires table, which has uniqueness constraint on type+dsl_version. There's a test for dsl_version.

Since there can be only 1 type in the DB, this test becomes unnecessary I think?